Weather/Scenes/Weather/WeatherPresenter.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  WeatherPresenter.swift
+//  Weather
+//
+//  Created by Vladislav Kondrashkov on 10/22/18.
+//  Copyright © 2018 Vladislav Kondrashkov. All rights reserved.
+//
+
+import UIKit
+
+protocol WeatherPresenter {
+    func updateData()
+    func updateCityLabel(cityLabel: UILabel)
```

Comment:
Again no UIKit inside presenters. You misinterpreted my idea about displaying data and notifying presenter from view.